bundles/org.openhab.automation.jrubyscripting/src/main/java/org/openhab/automation/jrubyscripting/internal/JRubyScriptEngineConfiguration.java

```diff
@@ -167,23 +178,29 @@ private synchronized void configureGems(ScriptEngine engine) {
 
             if (gem.isEmpty()) {
                 continue;
-            } else if (version.isEmpty()) {
-                gemCommand = "Gem.install('" + gem + "')\n";
-            } else {
-                gemCommand = "Gem.install('" + gem + "', '" + version + "')\n";
             }
 
-            try {
-                logger.debug("Installing Gem: {}", gem);
-                logger.trace("Gem install code:\n{}\n", gemCommand);
-                engine.eval(gemCommand);
-            } catch (ScriptException e) {
-                logger.warn("Error installing Gem: {}", e.getMessage());
-            } catch (BootstrapMethodError e) {
-                logger.warn("Error while checking/installing gems: {}. You may need to restart OpenHAB",
-                        e.getMessage());
-                logger.debug("Error in configureGems", e);
+            gemCommand += "  gem '" + gem + "'";
+            if (!version.isEmpty()) {
+                gemCommand += ", '" + version + "'";
             }
+            gemCommand += "\n";
```

Comment:
```suggestion
            gemCommand += ", require: false\n";
```

To maintain compatibility, we need to add `, require: false` here

Reply:
Good call.